src/main/java/pl/edukacyjni/user/UserController.java

```diff
@@ -1,11 +1,15 @@
 package pl.edukacyjni.user;
 
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
 
 import java.util.List;
+import java.util.Optional;
```

Comment:
there are some not used imports here - plese check "optimize imports" in intellij